docs/user/guides/connecting-git-account.rst

```diff
@@ -0,0 +1,22 @@
+How to connect your Read the Docs account to your Git repository
+================================================================
+
+If you are going to import repositories from GitHub, Bitbucket, or GitLab,
+you should connect your Read the Docs account to your repository host first.
+
+.. note::
+
+   If you signed up or logged in to Read the Docs with your GitHub, Bitbucket, or GitLab credentials,
+   you're all done. Your account is connected ✅️.
+
+To connect a social account, go to your :guilabel:`<Username dropdown>` > :guilabel:`Settings` > :guilabel:`Connected Services`.
```

Comment:
This feels underwhelming. We should have some other kind of content here, or a video? I think overall we should probably be sharing a couple benefits or something though, along with the basic process.

Reply:
I'll be doing a few updates on this one. I'm writing this comment to reflect on the goals of having a how-to here since it's border-line.

Goals of these changes:

* Showcase that it's _easy_ to add the integration
* Add screenshots that will make it easier to skim the how-to (because the texts are a bit trivial)
* Declare when this how-to is relevant